### ISSUE TITLE
Modify snow DA jobs to use Jinja templates to copy files

### ISSUE THREAD

**Issue by CoryMartin-NOAA** [NOAA-EMC/global-workflow#2736]
### What new functionality do you need?

FileHandler dictionaries are created in python currently, move this to Jinja2 templates for the snow DA job(s)

### What are the requirements for the new functionality?

See above

### Acceptance Criteria

CI test passes with the above completed

**Comment by WalterKolczynski-NOAA** [comment]
@CoryMartin-NOAA Was this completed?

**Comment by WalterKolczynski-NOAA** [comment]
@jiaruidong2017

**Comment by CoryMartin-NOAA** [comment]
I don't believe so, now that we have a `Jedi` class, this can be done as part of that refactoring

**Comment by CoryMartin-NOAA** [comment]
I believe this has been completed now
